decentraland/unity-explorer
# chore: move alpha_texture from PBRMaterial to UnlitMaterial

### WHY
Due to performance reasons the Explorer Alpha won't be supporting alpha texture on SDK PBR Materials any time soon.
However, the shader used for Unlit Material supports alpha texture.

### WHAT
Addresses the E@ part of [issue #240](https://github.com/decentraland/creator-hub/issues/240)

Related PRs:
Protocol: https://github.com/decentraland/protocol/pull/223
SDK: https://github.com/decentraland/js-sdk-toolchain/pull/1019

### QA TEST INSTRUCTIONS

1. Download the build from this PR
2. Go to AleAlvarez world: `/goto AleAlvarez`
3. There are 2 pillars, one is to show the alpha texture for video player on the OLD client, and one is for the NEW client. Click on both to check it works on the New Client (Old one is using PBR Material, New one is using BasicMaterial (Unlit)
